dapla-manual/faq.qmd

```diff
@@ -47,9 +47,15 @@ betyr det at mengden data som sendes fra jupyter-kernelen til jupyterlab-fronten
 
 ### Hvordan kan jeg gjenopprette filer fra bøtter? 
 
-Bøttene som opprettes i [featuren *dapla-buckets*](./statistikkere/features.html)^[Featuren *dapla-buckets* inkluderer produkt- og kildebøtta i henholdsvis standard- og kildeprosjektet som de fleste statistikkproduserende team får ved opprettelse.] har en *Retention policy* som gjør at slettede eller overskrevne objekter kan gjenskapes i en viss periode. Overskrevne filer blir såkalte *non-current versions* som lagres i 180 dager og deretter slettet. I tillegg vil *non-current* versjoner av en fil slettes hvis den har 2 nyere versjoner. 
+Bøttene som opprettes i [featuren *dapla-buckets*](./statistikkere/features.html)^[Featuren *dapla-buckets* inkluderer produkt- og kildebøtta i henholdsvis standard- og kildeprosjektet som de fleste statistikkproduserende team får ved opprettelse.] har en *Retention policy* som gjør at slettede eller overskrevne objekter kan gjenskapes i en viss periode.
 
-En fil som brukeren har slettet vil bli en *soft deleted* versjon og slettes etter 7 dager. 
+I dapla-buckets med objektversjoneringaktivert, blir overskrevne og slettede filer noncurrent.
+
+En noncurrent versjon blir slettet hvis den er mer enn 180 dager gammel og det er minst 2+ eller 3+ nyere versjoner i bøtta.
+
+(2+ for produktbøtte og 3+ for kildebøtte)
```

Comment:
Kan vi skrive dette som en fotnote? ^[I produktbøtta blir noncurrent versjoner slettet hvis det er mer enn 2 nyere versjoner, mens for kildebøtta er grensen på 3 nyere versjoner]